protocol/java/org/openyolo/protocol/CredentialDeleteResult.java

```diff
@@ -82,6 +84,12 @@
     /**
      * Pre-built deletion result for a bad request. Carries no additional properties.
      */
+    public static final CredentialDeleteResult UNKNOWN =
```

Comment:
1. Substitute documentation with: "Pre-built deletion result for an unspecified outcome. Carries no additional properties."
2. Rename variable to UNSPECIFIED to match pattern used by existing classes.